src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -160,17 +160,23 @@ public void clickGroupByTeam() {
         submitEditForm();
     }
 
-    public void clickCollapseExpandButton() {
+    public void clickCollapseExpandButtonAndWaitForPanelsToExpand() {
         click(collapseExpandButton);
+
```

Comment:
Now this method does not do as it's named...

Reply:
Will correct it

Reply:
Remember not to change any behaviour in any of the existing tests.

Reply:
Isn't it sufficient to change name to `clickCollapseExpandButton` ?